docs/tutorials/web/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploy on web
+parent: Tutorials
+has_children: true
+nav_order: 6
+---
+
+# How to add machine learning to your web application with ONNX Runtime
+
+ONNX Runtime Web enables you to run and deploy machine learning models in your web application using JavaScript APIs and libraries. This page outlines the general flow through the development process.
+
+To see an example of this flow in practice, you can follow the steps in the following tutorial to [build a web application to classify images using Next.js](classify-images-nextjs-github-template.md).
+
+For more detail on the steps below, see the [build a web application](../../reference/build-web-app.md) with ONNX Runtime reference guide.
+
+## ONNX Runtime web application development flow
+
+1. Choose deployment target and ONNX Runtime package
+
+   ONNX Runtime Web can be integrated into your web application in a number of different ways depending on the requirements of your application.
+
+   * Inference in browser. Use the `onnxruntime-web` package.
+
+     There are benefits to doing on-device and in-browser inference.
+
+     * **It's faster.** That's right, you can cut inferencing time way down which inferencing is done right on the client for models that are optimized to work on less powerful hardware.
+     * **It's safer** and helps with privacy. Since the data never leaves the device for inferencing, it is a safer method of doing inferencing.
+     * **It works offline.** If you lose internet connection, the model will still be able to inference.
+     * **It's cheaper.** You can reduce cloud serving costs by offloading inference to the browser.
+
+     You can also use the onnxruntime-web package in the frontend of an electron app.
+
+     With onnxruntime-web, you have the option to use `webgl` for GPU processing or WebAssembly (`wasm`) for CPU processing. All ONNX operators are supported by WASM but only a subset are currently supported by WebGL.
+
+   * Inference on server in JavaScript. Use the `onnxruntime-node` package.
+
+     Your application may have constraints that means it is better to perform inference server side.
+
+     * **The model is too large** and requires higher hardware specs. In order to do inference on the client you need to have a model that is small enough to run efficiently on less powerful hardware.
+     * You don't want the model to be downloaded onto the device.
+
+     You can also use the onnxruntime-node package in the backend of an elctron app.
+
+   * Inference on server using other language APIs. Use the `onnxruntime` packages for C/C++ and other languages.
```

Comment:
this is out of scope. but don't highlight "onnxruntime" as if it is a name of a NPM package here. This may confuse users. 

users don't need any NPM package in this scenario (they may try to look up to "onnxruntime" package in NPM repository, and what makes it worse is that there is such a package named "onnxruntime" -- the deprecated package for onnxruntime nodejs binding)

Reply:
Fixed